pkg/varnishcluster/controller/varnishcluster_grafana_dashboard_template.go

```diff
@@ -2397,8 +2403,9 @@ var grafanaDashboardTemplate = `
     ]
   },
   "timezone": "browser",
-  "title": "Varnish",
+  "title": "`+vc.Spec.Monitoring.GrafanaDashboard.Title+`",
```

Comment:
lets do it consistently and inject the the title as a template variable

Reply:
Do you mean passing `vc.Spec.Monitoring.GrafanaDashboard.Title` as a string parameter instead of passing the entire `vc` object?

Reply:
no, I meant to do the same as we do, for example for datasource name:
in the template: ` "datasource": "{{.DatasourceName}}",`.
And when we render the template we pass that variable [here](https://github.com/IBM/varnish-operator/pull/17/files#diff-63b182de2ceb1d598eb094bdcbb1c03c7cd412140e86921d374dd1e506d2fe3dR196)
No need to mix template generation and strings concatenation.

Reply:
I completely missed that, thanks